### ISSUE TITLE
Kalenderformatierung

### ISSUE THREAD

**Issue by tschortschde** [Zefau/ioBroker.jarvis#2405]
### Discussed in https://github.com/Zefau/ioBroker.jarvis/discussions/2404

<div type='discussions-op-text'>

<sup>Originally posted by **tschortschde** July 20, 2023</sup>
Hallo,
wäre es möglich das Widget anders zu formatieren?
Ich würde gerne 
- das Datum an einer anderen Stelle darstellen (z.B. vor der Uhrzeit)
- den Tag "Do.," in "Donnerstag," umbenennen 
- die Angabe "vor einem Tag" bzw. "in einem Tag" / "in etwa 3 Stunden" etc. ausblenden können
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/3b0424dd-b674-46e5-bc8a-ff95fa505d0c)

Ist das bzw. wie ist das umsetzbar?
Viele Grüße
</div>

**Comment by tschortschde** [comment]
Lässt sich das umsetzen?

**Comment by Zefau** [comment]
**Vote up your Feature Request!**

Vote up your feature request by selecting an emoticon **on the FIRST POST** (NOT THIS POST) as reaction:
_____
**Stimme für deinen Feature Request ab!**

Stimme für deinen Feature Requests ab indem du die Emoticons **beim ERSTEN BEITRAG** (NICHT DIESER BEITRAG) als Reaktion nutzt:
Siehe auch https://forum.iobroker.net/topic/30668/jarvis-v2-0-0-just-another-remarkable-vis/1986

![Github Reactions](https://forum.iobroker.net/assets/uploads/files/1606215967286-1c3e566f-8d54-4f19-8948-eb8833c5b15f-image.png)

**Comment by tschortschde** [comment]
Möchte nochmal nachfrage, ob das umsetzbar ist?

**Comment by Zefau** [comment]
Ja, natürlich. Alles ist möglich. Ich muss es nur in der Konfiguration einbauen.

**Comment by tschortschde** [comment]
Bis wann kann das umgesetzt werden?  😃

**Comment by Zefau** [comment]
Hat nur wenig votes und andere Sachen erscheinen mir wertvoller
